### PR TITLE
Review

### PR DESCRIPTION
**Pull request by kru**
#15 
#16 
#17 
#18 

screen shot 2018-02-25 at 19 23 56

### CODE REVIEW

File: mobile/src/redux/modules/review.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+export const WRITE_REVIEW_ACTIONS = Object.freeze({
+  IS_LOADING: `${CONFIG.APP_NAME}/write-review/isLoading`,
+  FETCH_REVIEW: `${CONFIG.APP_NAME}/write-review/fetchReview`,
+  FULFILLED: `${CONFIG.APP_NAME}/write-review/fulfilled`,
+  REJECTED: `${CONFIG.APP_NAME}/write-review/rejected`,
+})
+
+export const writeReviewActions = Object.freeze({
+  isLoading: status => ({ type: WRITE_REVIEW_ACTIONS.IS_LOADING, status }),
```
Comment (AaronVasquez) [inline mobile/src/redux/modules/review.js]: You don't need this action. When you receive a `FETCH_REVIEW` action, you can set loading to `true`, and when you receive a `FULFILLED` or `REJECTED`, you can set loading to `false`.

It's good practice to have as few actions as possible.

File: mobile/src/redux/modules/review.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import CONFIG from '../../../config'
+
+export const WRITE_REVIEW_ACTIONS = Object.freeze({
+  IS_LOADING: `${CONFIG.APP_NAME}/write-review/isLoading`,
```
Comment (AaronVasquez) [inline mobile/src/redux/modules/review.js]: Nit: keep naming kebab `is-loading`, `fetch-review`, etc.

File: mobile/src/redux/effects/reviews-fx.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { ofType } from 'redux-observable'
```
Comment (AaronVasquez) [inline mobile/src/redux/effects/reviews-fx.js]: This file belongs in the `/epics` folder. `effects` is for actually doing the API call. I'll guide you with more directions in a task when we start working on the API.

File: mobile/src/redux/selectors/review-selector.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { createSelector } from 'reselect'
+
+const getReviewReducer = state => state.review
+
+export const getReview = createSelector(
```
Comment (AaronVasquez) [inline mobile/src/redux/selectors/review-selector.js]: These don't need to be selectors. We use `reselect` for derived state, like sorting or filtering the list of reviews, but not just to pluck pieces of state. (Let me know if you need me to explain this more)

File: mobile/src/redux/effects/reviews-fx.js (hunk 3 above)
Comment (kru) [reply mobile/src/redux/effects/reviews-fx.js]: Curious about this`epics` and `effects` implementation, but You could later explain to me when We start working on API.

File: mobile/src/redux/selectors/review-selector.js (hunk 4 above)
Comment (kru) [reply mobile/src/redux/selectors/review-selector.js]: So this means our store keep the data as is, then all data transformations should be done in selector?

**Pull request merged**